api/services/LocationService.js

```diff
@@ -44,17 +46,13 @@ var LocationService = {
     },
 
     postcodeLookup: function postcodeLookup(postcode) {
-      var rp = require('request-promise');
       var options = JSON.parse(JSON.stringify(sails.config.customURLs.postcodeLookUpApiOptions));
```

Comment:
probably some old code? This is the old way to clone an object (nowadays you would just do `const = { ...myObj }`. But I can't see why it needs to be cloned tbh.. so suggestion👇 cos it's not your problem (but it's bugging me too much I want to fix it lol) 

```suggestion
     const { uri, timeout } = sails.config.customURLs.postcodeLookUpApiOptions;
     return axios.get(`${ uri }${ postcode }`, { timeout })
```